addons/mail/static/src/core/web/chatter.js

```diff
@@ -199,16 +202,24 @@ export class Chatter extends Component {
      */
     get toFollowersText() {
         const threadName = this.state.thread.displayName || this.state.thread.name;
+        const followers = [...this.state.thread.followers]
+            .reverse()
+            .slice(-5)
+            .map(({ partner }) =>
+                sprintf('<span title="%(email)s">%(emailWithoutDomain)s</span>', {
+                    email: escape(partner.email),
+                    emailWithoutDomain: escape(partner.emailWithoutDomain),
+                })
+            );
+        const formatter = new Intl.ListFormat(
+            this.store.env.services["user"].lang?.replace("_", "-"),
+            { type: "conjunction" }
+        );
```

Comment:
If there are more than 5 recipients, conjunction should have `and more` at the end, otherwise it's misleading:
<img width="1058" alt="Screenshot 2023-06-28 at 12 07 59" src="https://github.com/odoo/odoo/assets/6569390/0fa15d35-98c2-4ddf-b45c-8a675669c3da">
7 recipients, but inline listing gives impression there are only 5 recipients

Reply:
IMO, having more than 5 recipients should display "..." at the end of the list, because "and more" is a little too long 
At the moment, there is an unexpected "and" before the 5th recipient, even if there are more than 5.
![image](https://github.com/odoo/odoo/assets/124038285/703a3e22-2c69-4566-91f6-edf5698ed343)


@phenix-factory @alexkuhn

Reply:
The "and" is the result of using `Intl.ListFormat` that think the list is complete. I fixed that with a work around.